cfgov/unprocessed/css/pages/event.less

```diff
@@ -191,8 +198,33 @@
     }
 }
 
+.event-venue_header {
+    margin-bottom: 10px;
+}
+
+.btn.event-venue_live-btn {
+    margin-right: 10px;
+    background-color: @green !important;
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/pull/1625/commits/36f6195becc4c28009ba7b84c31290ffe07277ae